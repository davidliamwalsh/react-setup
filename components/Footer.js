import { Component } from 'react'

class Footer extends Component {
  render () {
    return <div>
      <ul>
        <li><a href='#'>Footer Link</a></li>
      </ul>
    </div>
  }
}

export default Footer