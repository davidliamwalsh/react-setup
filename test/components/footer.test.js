import Footer from '../../components/Footer'

describe('<Footer />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <Footer />
    )

    expect(component).toMatchSnapshot()
  })
})