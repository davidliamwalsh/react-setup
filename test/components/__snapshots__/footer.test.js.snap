// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> it renders correctly 1`] = `ShallowWrapper {}`;
